app/services/combined_audio_derivative_creator.rb

```diff
@@ -0,0 +1,163 @@
+require 'tempfile'
+#
+# Generate the combined audio derivatives for a
+# given oral history work.
+#
+# some_work = Work.find_by_friendlier_id(friendlier_id)
+# CombinedAudioDerivativeCreator.new(some_work).generate
+#
+#
+# Sample output:
+#     {
+#      :combined_audio_mp3_data=>
+#          "/var/folders/n7/t4zt2w751bj_6pbnf95sgpnrmz8lz8/T/output20200228-61637-ec6kjz.mp3",
```

Comment:
Please return a ruby TempFile, not a file path.

Reply:
Done.